act-rules-format.bs

```diff
@@ -83,58 +83,43 @@ Even within a rules group, certain rules may not be used. This leaves one fewer
+The means by which the DOM tree is constructed, be it by a web browser or not, is not of importance as long as the construction follows any applicable specifications that might exist, such as [[DOM]].
 
+### CSS Styling ### {#input-aspects-css}
 
-Rendered Page Testing {#input-types-rendered-page}
---------------------------------------------
+The computed CSS styling resulting from parsing CSS and applying it to the DOM may be of interest to ACT Rules that wish to test the web page as presented to the user. Through CSS styling, information about the position, the fore- and background colors, the visibility, and more, of elements becomes available.
 
-Testing the browser is the next level up from DOM Tree Testing. In addition to building the DOM Tree, the browser styles elements in the DOM tree and positions them. This enables a rule to determine if an element is visible, which is critical for many tests. Additionally, testing things like the color contrast becomes possible at this level.
+The means by which the CSS styling is computed, be it by a web browser or not, is not of importance as long as the computation follows any applicable specifications that might exist, such as [[CSSOM]].
 
+### Accessibility Tree ### {#input-aspects-accessibility}
 
-Template Testing {#input-types-template}
-----------------------------------
+The accessibility tree constructed from extracting information from both the DOM tree and the CSS styling may be of interest to ACT Rules that wish to test the web page as presented to assistive technologies such as screen readers. Through the accessibility tree, information about the semantic roles, accessible names and descriptions, and more, of elements becomes available.
 
-A template is a document that has "open" fields that are filled with pieces of content or other templates. E.g. an HTML template contains a head with a variable title, a predefined navigation, and a variable content area.
+The means by which the accessibility tree is constructed, be it by a web browser or not, is not of importance as long as the construction follows any applicable specifications that might exist, such as [[CORE-AAM-1.1]].
 
-<div class=example>
-`<img>` tags with a variable `src`, MUST NOT have a static `alt` value.
-</div>
+### Language ### {#input-aspects-text}
 
+Language, either written or spoken, as available in e.g. nodes of the DOM or accessibility trees may be of interest to ACT Rules that wish to test things like complexity or intention of the language. For example, an ACT Rule might wish to test that certain paragraphs of text within the DOM tree do not exceed a certain readability score or that the text alternative of an image provides a sufficient description thereof.
```

Comment:
The "e.g." in the middle of the first sentence makes it read a bit awkwardly.
Again, the use of "wish to".  And just wondering, how do you test the "intention" of the language?  How do you test intent?